Foundation/NSLocale.swift

```diff
@@ -13,9 +13,9 @@ import CoreFoundation
 public class NSLocale : NSObject, NSCopying, NSSecureCoding {
     typealias CFType = CFLocale
     private var _base = _CFInfo(typeID: CFLocaleGetTypeID())
```

Comment:
A general warning for the future: Swift property layout is not guaranteed to be the same as C property layout. I don't actually know of a safe way to do this, but the safest way there is today is to declare all the properties in a C struct and have a single property Swift-side of that type.